src/scene_battle_rpg2k3.cpp

```diff
@@ -980,7 +980,7 @@ bool Scene_Battle_Rpg2k3::CheckWin() {
 		std::vector<int> drops;
 		Main_Data::game_enemyparty->GenerateDrops(drops);
 
-		Game_Message::texts.push_back(Data::terms.victory);
+		Game_Message::texts.push_back(Data::terms.victory + "\\|");
 
```

Comment:
> 2.7: RM2000 + RM2003: The Victory message should have pauses after...

I'm almost sure that these "\\"-codes must be replaced with Player::escape_symbol, otherwise they won't be recognized in japanese or korean games.